Blockzilla/BrowserViewController.swift

```diff
@@ -1077,6 +1088,20 @@ extension BrowserViewController: HomeViewDelegate {
         let shareController = UIActivityViewController(activityItems: [text, appStoreUrl as Any], applicationActivities: nil)
         present(shareController, animated: true)
     }
+    
+    func tipTapped() {
+        guard let tip = tipManager?.currentTip, tip.showVc else { return }
+        switch tip.identifier {
+        case TipManager.TipKey.biometricTip, TipManager.TipKey.siriEraseTip:
+            showSettings()
+        case TipManager.TipKey.siriFavoriteTip:
+            if #available(iOS 12.0, *) {
```

Comment:
I construct it so that this tip never gets added to the possible tips in the TipManager. However, because the SiriFavoriteVC is only @/available for iOS 12, the compiler requires an additional check. Users should never have the option to click on this tip (because it will never be possible)

Reply:
Ah, it's an annoying compiler thing; I was wondering why we were checking twice. Makes sense!